src/commandHandlers/help.ts

```diff
@@ -10,24 +10,26 @@ export const command = async (arg: [string, string], embed: MessageEmbed) => {
     triggers.find((trigger) => trigger === arg[1])
   );
 
-  embed.setTitle('Help commands').setDescription('Lists the command available');
-
   if (matching) {
-    embed.addField(
-      `${COMMAND_PREFIX}${matching.triggers[0]}`,
-      `${matching.description}\nUsage: ${COMMAND_PREFIX}${matching.usage}`,
-      false
-    );
+    const blocky = (arg: string) =>
+      '`' + `${COMMAND_PREFIX}` + `${arg}`.trim() + '`';
+    const text = `${matching.usage}`.split('or');
+    const formatted = text.map(blocky).join(' or ');
+
+    embed
+      .setTitle(`${COMMAND_PREFIX}${matching.triggers[0]}`)
+      .setDescription(
+        `${matching.description}` + '\nUsage:\t' + `${formatted}`
+      );
   } else {
-    let text = '';
-    commandList.forEach((commandItem) => {
-      text = text
-        .concat(`**${COMMAND_PREFIX}${commandItem.triggers[0]}**  `)
-        .concat(`${commandItem.description}\n`)
-        .concat(
-          'Usage:' + '`' + `${COMMAND_PREFIX}${commandItem.usage}\n` + '`'
-        );
-    });
+    embed
+      .setTitle('Help commands')
+      .setDescription(
+        'These are the available commands. For more information on a specific command, use `^help <commandName>`'
+      );
+    const format = (commandItem: CommandHandler) =>
+      '`' + `${COMMAND_PREFIX}${commandItem.triggers[0]}` + '`';
+    const text = commandList.map(format).join(' | ');
```

Comment:
Then here we can use the same function, since I think trimming the trigger of each command item is fine. But let me know if I'm missing something
```suggestion
    const formatWithCodeBlocks = (commandItem: CommandHandler) => mapToCodeBlock(commandItem.triggers[0]);
    const text = commandList.map(mapToCodeBlock).join(' | ');
```